.travis.yml

```diff
@@ -22,7 +22,7 @@ matrix:
       script: npm run pretest
       env: CI=pretest
     - node_js: '8'
-      script: npm run jest -- --runInBand  --coverage
+      script: npm run jest -- --runInBand --coverage
```

Comment:
This change should also be removed, it should be its own pull request please

Note: _Pull requests should only ever contain the relevant changes required for the intended changes of the pull request._